rust/ql/lib/codeql/rust/elements/internal/VariableImpl.qll

```diff
@@ -366,6 +387,29 @@ module Impl {
     override string getAPrimaryQlClass() { result = "VariableAccess" }
   }
 
+  /** Holds if `e` occurs in the LHS of a (compound) assignment. */
```

Comment:
I think we mean a compound or non-compound assignment, but the brackets aren't communicating that to me clearly.
```suggestion
  /** Holds if `e` occurs in the LHS of an assignment or compound assignment. */
```

Reply:
(addressed)